classes/session/SessionDAO.inc.php

```diff
@@ -192,6 +193,22 @@ public function sessionExistsById($sessionId)
         $row = $result->current();
         return $row ? (bool) $row->row_count : false;
     }
+
+	/**
+     * Delete given user's all sessions or except for the given session id
+     *
+     * @param int                   $userId
```

Comment:
If the annotations are only reiterating the type hints, better to leave them out. If you want to include parameter descriptions (and I recommend it), unfortunately it's currently necessary to restate the type. (Ref for `@param` PHPDoc tag: https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc-tags.md#510-param)